client/static/js/hatohol_monitoring_view.js

```diff
@@ -19,6 +19,7 @@
 
 window.onerror = function(errorMsg, fileName, lineNumber) {
   var place = "[" + fileName + ":" + lineNumber + "]";
+  console.error(place + " " + errorMsg);
```

Comment:
I noticed that sometimes trailing async functions overwrite the previous status and discard 
the previous error messages immediately (when a trailing async function doesn't have errors, 
the status becomes "DONE" and the button becomes green). It's hard to debug. 

So I want to output all error messages to console.